Xamarin.Forms.Platform.UAP/FormsComboBox.cs

```diff
@@ -7,17 +7,35 @@ namespace Xamarin.Forms.Platform.UWP
 {
 	public class FormsComboBox : ComboBox
 	{
-		internal bool IsClosingAnimated { get; private set; }
+		public FormsComboBox()
+		{
+			DefaultStyleKey = typeof(FormsComboBox);
+			DropDownOpened += FormsComboBoxDropDownOpened;
+			SelectionChanged += FormsComboBoxSelectionChanged;
+		}
 
-		internal bool IsFullScreen => Device.Idiom == TargetIdiom.Phone && Items != null && Items.Count > 5;
+		~FormsComboBox()
+		{
+			DropDownOpened -= FormsComboBoxDropDownOpened;
```

Comment:
These aren't needed

Reply:
I wondered about that...I'll remove them